week-4/C-more-complex-objects/exercise-1.js

```diff
@@ -0,0 +1,33 @@
+/*
+  Given the following house - follow the instructions below.
+
+  Make sure you run the file after and it outputs the correct results.
+*/
+
+var house = {
+    address: '1 Kinning Park',
+    previousOwners: ["Claire M.", "John A."],

```

Comment:
In this exercise you mix `"` and `'`. I'm not sure which we're using. Maybe check with Daniel.

Reply:
I think the the double quotes have been the more popular, and yes, either way, I should keep it consistent and use only one type of quotes

Reply:
Yep. If you have prettier installed and format on save set it should take care of it for you (even in markdown)